src/translations/pages/contact.ts

```diff
@@ -52,7 +53,7 @@ export const Contact = {
         ),
 
         app: {
-          title: _('App', 'App', 'App'),
+          title: _('Mobilapp', 'Mobile App', 'Mobilapp'),
```

Comment:
Should the 'a' in "Mobile app" be capitalized?

Reply:
No, it should not. Will fix now👍